Framework/PythonInterface/plugins/algorithms/LRAutoReduction.py

```diff
@@ -425,7 +426,7 @@ def _get_direct_beam(self, meta_data_run, data_set):
         direct_beam_runs_str = self._read_property(meta_data_run, "direct_beam_runs",
                                                    _direct_beam_runs, is_string=True)
         # The direct runs in the DAS logs are stored as a string
-        if isinstance(direct_beam_runs_str, (str, unicode)):
+        if isinstance(direct_beam_runs_str, str):
```

Comment:
I just found this:

```
from six import string_types

if isinstance(direct_beam_runs_str, string_types):
    ....
```
But haven't tested it. From the docs:

"six.string_types: Possible types for text data. This is basestring() in Python 2 and str in Python 3."